content/en/docs/how-tos/naming-your-ci-jobs.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Naming your CI Jobs"
+description: How to name your CI job.
+---
+
+## Job naming guidelines
+
+Job names should indicate to the reader the purpose of the specific job,
+as well as noting where the job deviates from OpenShift defaults for
+that release. The guidelines contained in this document are based on the
+way the majority of jobs are named today. The general format of a job
+name should be as follows:
+
+```
+[JOB TYPE]-ci-[GITHUB ORG]-[GITHUB REPO]-[GITHUB BRANCH]-[STREAM]-[RELEASES]-[TEST SUITE]-[PLATFORM]-[CNI]-[OTHER DEVIATIONS]
+```
+
+When using multi-stage generated jobs, the first part of the name is
+generated for you, and the second part that begins with "releases" is
+determined by the job creator.
+
+## Job name fields
+
+### Generated fields
+
+For multi-stage jobs, these fields are generated for you and aren't
+needed in the name set in your YAML configuration.
+
+**Job type**:  Will be either "periodic" or "pull". Jobs that begin with
+other prefixes like "release" or  "promote" are template-style jobs and
+mostly not used any more.
+
+**GitHub org/repo/branch**: The GitHub org, repo and branch the job is
+configured for.  Release periodics typically use openshift/release, but
```

Comment:
what is the branch for a periodic?

Reply:
They still get assigned to a branch, for openshift/release they're attached to master, and use ci-opreator variants for each release - https://github.com/openshift/release/tree/master/ci-operator/config/openshift/release

Hypershift and a few others tie them to the release-X.Y branch for that periodic.